lib/beaker/options/parser.rb

```diff
@@ -228,10 +228,22 @@ def parse_hosts_options
         hosts_options = Beaker::Options::HostsFileParser.parse_hosts_file(@options[:hosts_file])
         return hosts_options
       rescue Errno::ENOENT
-        require 'tempfile'
+        error_message = "\nHosts file '#{@options[:hosts_file]}' does not exist."
+        error_message << "\nTrying as beaker-hostgenerator input.\n\n"
+        $stderr.puts error_message
```

Comment:
At this point, we can't know if this is going to be an error or not. There's only one code path that runs through these print statements where no error has occurred (correct beaker-hostgenerator syntax that was intended). Since there are a number of other pathways through here, and those are all error cases (examples below), I believe using standard error here is correct.

- host file path improperly specified
- beaker-hostgenerator syntax incorrect

One other option we could use here is to wait and not print until we know a little more. Since if the beaker-hostgenerator use succeeds, we don't typically print out at all, it seems that just deleting this might be a better option. However, my concerns with this are that:

1. If you checkout the logging/printing that happens above in my snippet in the first comment, you'll see that a beaker-hostgenerator warning is issued. Without the printing done here, there's no context for that warning, and beaker users will be confused about where this is coming from and why it's there without the additional printing.
2. If there's either something wrong with the importing mechanism (doubtful) or with the way beaker initializes and uses beaker-hostgenerator (not as surprising), then this will fail with a potentially uncaught exception type, with no output that even let the beaker user know they were using a different application to generate their hosts file at all.

I'm of the opinion that this should be kept. @nwolfe what do you think?